docs/release-notes/mongoid-8.1.txt

```diff
@@ -18,6 +18,31 @@ please consult GitHub releases for detailed release notes and JIRA for
 the complete list of issues fixed in each release, including bug fixes.
 
 
+Configuration DSL No Longer Requires an Argument to its Block
+-------------------------------------------------------------
+
+In Mongoid 8.1, it is now possible to use ``Mongoid.configure`` without
```

Comment:
Please do not add version numbers to release notes like this, it makes backporting changes more difficult and the release notes already specify the version they are for.